lib/Benchmark/SamplerManager.php

```diff
@@ -55,7 +56,7 @@ public function addSamplerCallable(string $name, $callable): void
             throw new InvalidArgumentException(sprintf(
                 'Given sampler "%s" callable "%s" is not callable.',
                 $name,
-                is_string($callable) ? $callable : gettype($callable)
+                is_string($callable) ? $callable : gettype($callable) // @phpstan-ignore-line
```

Comment:
no need to change, but just for information I found out about [get_debug_type](https://www.php.net/manual/en/function.get-debug-type.php) which is better than `gettype` here.

Reply:
Yes, `get_debug_type` is cool. But I think it breaks B/C as some users can rely on Exception message. So it can't be included into minor/patch release.
With `get_debug_type` the message will look like this if `$callable === 'bar'`: `Given sampler "foo" callable "string" is not callable.`
And with gettype the message looks like this: `Given sampler "foo" callable "bar" is not callable.`

Reply:
i hope users don't rely on the exception message, but valid point :)